VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKRefund.h

```diff
@@ -0,0 +1,28 @@
+#import "VSDKTrackingEvent.h"
+@class VSDKTransaction;
+@class VSDKProduct;
+
+NS_ASSUME_NONNULL_BEGIN
+
+//TODO Split this into VSDKPartialRefund and VSDKTotalRefund
+@interface VSDKRefund : VSDKTrackingEvent
+
+/** 
+  Type of refund 
+  A refund can be either `partial` or `total`
+ */
+@property NSString *refundType;
+
+/**
+  If it is a `partial` refund, the list of products to be refunded must be specified
```

Comment:
Because we will probably redo this, I prefer to leave this change for a future PR.

Reply:
Are you referring to the TODO above? Unless there is a concrete task for us to do it in the future, I don't know when it will be done. Also, that TODO isn't a restriction for us to ship the SDK, right? Why not implement this logic right away? We will be more relaxed to ship this release and the user will have more trouble to create an invalid event.

Reply:
> Are you referring to the TODO above?

Yes. 

> Unless there is a concrete task for us to do it in the future, I don't know when it will be done. Also, that TODO isn't a restriction for us to ship the SDK, right? Why not implement this logic right away? We will be more relaxed to ship this release and the user will have more trouble to create an invalid event.

I prefer to leave the model classes a bit crude because they are not the main focus of this PR and we can iterate on that later and add other quality of life improvements, like proper initializers (to avoid using the default `init` and enforce the initialization of the required properties). As it is, this PR is already really big and I would prefer to have changes that add functionality and can be packed to come in a future PR.